demosplan/DemosPlanCoreBundle/Entity/Procedure/ProcedureSettings.php

```diff
@@ -202,28 +202,28 @@ class ProcedureSettings extends CoreEntity implements UuidEntityInterface, Proce
      *
      * @ORM\Column(name="_ps_designated_phase", type="string", length=50, nullable=true)
      */
-    protected $designatedPhase = null;
```

Comment:
I would not remove the 'null' initialization as it is not a default value in the ORM sense, but might also be needed as a fallback for the values.